.node-version

```diff
@@ -0,0 +1 @@
+6.10.2
```

Comment:
Pinning to a specific patch release seems odd for a utility library. Upgrades to node will mean avn isn't as useful unless we keep this updated.

What about using `^6.0.0` or even `>=6.0.0`? That's what we're checking for when logging an error anyway...